From 36b113973570794444e71e388e3a32105ca05234 Mon Sep 17 00:00:00 2001
From: Tsuyoshi Ozawa <ozawa@apache.org>
Date: Thu, 22 Jan 2015 12:25:05 +0900
Subject: [PATCH 2505/2848] YARN-3078. LogCLIHelpers lacks of a blank space
 before string 'does not exist'. Contributed by
 Sam Liu.

(cherry picked from commit 5712c9f96a2cf4ff63d36906ab3876444c0cddec)

Change-Id: Ib89f6029b4158ce617a4f6a7c9dde18c8d0d37b0
---
 .../hadoop/yarn/logaggregation/LogCLIHelpers.java  |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/logaggregation/LogCLIHelpers.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/logaggregation/LogCLIHelpers.java
index 9442e23..ac14f22 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/logaggregation/LogCLIHelpers.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/logaggregation/LogCLIHelpers.java
@@ -226,11 +226,11 @@ private static void containerLogNotFound(String containerId) {
   }
 
   private static void logDirNotExist(String remoteAppLogDir) {
-    System.out.println(remoteAppLogDir + "does not exist.");
+    System.out.println(remoteAppLogDir + " does not exist.");
     System.out.println("Log aggregation has not completed or is not enabled.");
   }
 
   private static void emptyLogDir(String remoteAppLogDir) {
-    System.out.println(remoteAppLogDir + "does not have any log files.");
+    System.out.println(remoteAppLogDir + " does not have any log files.");
   }
 }
-- 
1.7.9.5

